charts/extended-daemon-set/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for extendeddaemonset.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+image:
+  repository: datadog/extendeddaemonset
+  tag: v0.6.0
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+logLevel: "info"
+# Allow ExtendedDaemonset controller to watch all namespaces
+# Defaulted to false
+clusterScope: false
+pprof:
+  enabled: false
+rbac:
+  # Specifies whether the RBAC resources should be created
+  create: true
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+podSecurityContext: {}
+# fsGroup: 2000
+
+securityContext: {}
+# capabilities:
+#   drop:
+#   - ALL
+# readOnlyRootFilesystem: true
+# runAsNonRoot: true
+# runAsUser: 1000
+
+resources: {}
+# We usually recommend not to specify default resources and to leave this as a conscious
+# choice for the user. This also increases chances charts run on environments with little
+# resources, such as Minikube. If you do want to specify resources, uncomment the following
+# lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+# limits:
+#   cpu: 100m
+#   memory: 128Mi
+# requests:
+#   cpu: 100m
+#   memory: 128Mi
+
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+datadog-crds:
+  crds:
+    # datadog-crds.crds.extendedDaemonSets -- Set to true to deploy all the EDS CRDs
+    # (ExtendedDaemonSet, ExtendedDaemonSetReplicaSet, ExtendedDaemonSettings)
+    extendedDaemonSets: true
```

Comment:
I don't think the comments format is compatible with the [doc generation script](https://github.com/DataDog/helm-charts/blob/master/.github/helm-docs.sh)  - see [this](https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml) as example.

It should be 

```
# field1 -- <doc string>
field1: val1

field2:
  # field2.field2-1 -- <doc string>
  field2-1: val2-1
```

Like this `.github/helm-docs.sh` will be able to update the readme automatically

Reply:
As it is right now the script generated:
```
| datadog-crds.crds.extendedDaemonSets | bool | `true` | Set to true to deploy all the EDS CRDs (ExtendedDaemonSet, ExtendedDaemonSetReplicaSet, ExtendedDaemonSettings) |
```

If I change this to:
```
# datadog-crds -- Datadog CRDs
datadog-crds:
  crds:
    # crds.extendedDaemonSets -- Set to true to deploy all the EDS CRDs
    # (ExtendedDaemonSet, ExtendedDaemonSetReplicaSet, ExtendedDaemonSettings)
    extendedDaemonSets: true
```

The script generates:
```
| datadog-crds | object | `{"crds":{"extendedDaemonSets":true}}` | Datadog CRDs |
```

Reply:
you should only comment the leafs 
```
datadog-crds:
  crds:
    # crds.extendedDaemonSets -- Set to true to deploy all the EDS CRDs
    # (ExtendedDaemonSet, ExtendedDaemonSetReplicaSet, ExtendedDaemonSettings)
    extendedDaemonSets: true
```

Reply:
my comment was mainly about the other fields in this fiels, most of them are missing the doc comment

Reply:
You're right. I added proper comments for all of them 👍